package.json

```diff
@@ -21,7 +21,7 @@
 		"test": "npm run lint && npm run jsdoc-generate && npm run coverage",
 		"test-azure": "npm run lint && npm run jsdoc-generate && npm run coverage-xunit",
 		"lint": "eslint ./",
-		"unit": "rimraf test/tmp && ava",
+		"unit": "rimraf test/tmp && ava -v",
```

Comment:
That's what the `unit-verbose` npm script one line below is supposed for. Or was this changed for some temporary test?

Reply:
was just temporary, this change somehow slided in, will revert this line